public/lessons/javascript/encapsulation.md

```diff
@@ -108,12 +108,7 @@ Note that only `area` and `circumference` are exposed via a pointer to the circl
 
 Expanded:
 
-```
-( function() { ... }  ) ();
-   ↑           ↑         ↑
-  define       ↑       invoke
-              execute   
-```
+![IIFE Example](/images/iife-breakdown.png)
```

Comment:
@Habenzy This file does not exist. Maybe it needs to be uploaded? Or the name is different?

Reply:
I just checked this on the PR deployment and it is fine.